packages/components/src/components/dropdown/dropdown.scss

```diff
@@ -300,4 +308,23 @@
             right: 16px;
         }
     }
+    &--filter {
+        margin-left: 1em;
+        margin-top: 0;
+        min-width: 3rem;
+        width: 3rem;
+
+        @include mobile {
+            height: 4em;
+        }
+    }
+}
+
+.icon_filter {
+    position: absolute;
+}
+
+.dc-list-filter {
+    width: 100px !important;
+    left: -70px;
```

Comment:
QS: Will this rule produce the same behavior on different screen sizes?

Reply:
yes already tested